packages/myst-frontmatter/src/frontmatter/types.ts

```diff
@@ -25,41 +25,32 @@ export type Biblio = {
 };
 
 export type Thebe = {
-  useBinder?: boolean;
-  useJupyterLite?: boolean;
-  requestKernel?: boolean;
-  binderOptions?: ThebeBinderOptions;
-  serverSettings?: ThebeServerSettings;
-  kernelOptions?: ThebeKernelOptions;
-  savedSessionOptions?: ThebeSavedSessionOptions;
-  mathjaxConfig?: string;
+  lite?: boolean;
+  binder?: boolean | ThebeBinderOptions;
+  server?: boolean | ThebeServerOptions;
```

Comment:
Should `server` be allowed to be a boolean? It can't really be `true`; we have to know the URL. And if it's `false`, just leave it undefined.

Reply:
hmm.... the only `server: true` thing that makes sense if it also defaults to the same configuration as `local`...
not sure we should do that though?

Reply:
I see, makes enough sense to leave it!